build/scripts/PreCompileApp.ps1

```diff
@@ -69,3 +77,8 @@ if($appType -eq 'app')
         }
     }
 }
+
+if($useCompilerFolder) {
```

Comment:
Not sure why this is needed. The assembly probing paths are set when using a compiler folder.

Reply:
It was needed because containerhelper automatically would pick .NET 9 as an assemblyproblingpath. I'm guessing that was fixed with this change: https://github.com/microsoft/navcontainerhelper/commit/c846102da530b72ebc0e30de026abfc938363b02

So we might not need this anymore